scripts/issue/1176/badger_wbtc_bunni_deposit.py

```diff
@@ -0,0 +1,29 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+from pycoingecko import CoinGeckoAPI
+
+
+BADGER_USD = 250_000
+
+
+def main():
+    vault = GreatApeSafe(r.badger_wallets.treasury_vault_multisig)
+    vault.init_bunni(r.bunni.badger_wbtc_bunni_token)
+
+    badger = vault.contract(r.treasury_tokens.BADGER)
+    wbtc = vault.contract(r.treasury_tokens.WBTC)
+    bunni_gauge = vault.contract(r.bunni.badger_wbtc_bunni_gauge)
+
+    vault.take_snapshot(tokens=[badger, wbtc, bunni_gauge])
+
+    prices = CoinGeckoAPI().get_price("badger-dao", "usd")
+    badger_price = prices["badger-dao"]["usd"]
+
+    amount_badger = (BADGER_USD / badger_price) * 1e18
```

Comment:
in the snapshot this seems to be undershooting for the target ticks interval

<img width="790" alt="Screenshot 2023-03-16 at 18 36 50" src="https://user-images.githubusercontent.com/84875062/225626321-43c875a2-a1ce-49bb-9903-cfc4b00d15c0.png">

we will be depositing ~133k in wbtc and ~245k in badger. totalling ~$378k instead of half M

in this case the ticks interval is expecting to deposit more amount of badger in dollar terms (>250k) compare to the wbtc

Reply:
<img width="1167" alt="Screenshot 2023-03-16 at 18 48 18" src="https://user-images.githubusercontent.com/84875062/225629039-a6f8cf1a-5e72-4db8-bc56-7e308451cac9.png">

Reply:
yea we need to get ratio of start tick, end tick and current tick in order to calc (rough) deposit ratio

Reply:
```
lower = vault.bunni.bunni_key.tickLower
upper = vault.bunni.bunni_key.tickUpper
current = Contract(vault.bunni.bunni_key.pool).slot0()[1]
full_range = upper - lower
upper_range = upper - current
ratio = upper_range / full_range
```
```
amount_badger = BADGER_USD * (1 + ratio) / badger_price * 1e18
```

Reply:
(assuming we are in range)

Reply:
solution in 91c0c8c

Reply:
feels like with `get_amounts_for_liquidity` present it should be even easier to approach the amounts mathematically, instead of approach them like you do through that recursive func.

gets the job done though, lets go!

Reply:
ya that solution was a bit dirty, updated to a more simple and precise calc: ac40989

Reply:
yes looking better now, also much more reusable for the future :)